packages/adapter-node/src/index.js

```diff
@@ -31,7 +33,16 @@ let shutdown_timeout_id;
 /** @type {NodeJS.Timeout | void} */
 let idle_timeout_id;
 
-const server = polka().use(handler);
+const httpServer = http.createServer();
+const server = polka({ server: httpServer }).use(handler);
+
+const ws = crossws({
+	resolve
+});
+
+httpServer.on('upgrade', (req, socket, head) => {
+	ws.handleUpgrade(req, socket, head);
+});
```

Comment:
Polka already uses `http.createServer()` by default. We can access the http server instance via the `server` property.
```suggestion
const server = polka().use(handler);

const ws = crossws({
	resolve
});

server.server.on('upgrade', (req, socket, head) => {
	ws.handleUpgrade(req, socket, head);
});
```

Reply:
Thank you!

Their docs had me a bit confused as to why separate servers were created for some examples.